test/unit/browser/vscode.test.ts

```diff
@@ -0,0 +1,88 @@
+/**
+ * @jest-environment jsdom
+ */
```

Comment:
A little trick I learned from StackOverflow! This means no `Location is not defined` errors and you can write your tests peacefully!

Reply:
Neat!!